packages/sdk/src/sdk-blockchains/solana/index.ts

```diff
@@ -18,7 +18,7 @@ export function createSolanaSdk(
 	cluster: Cluster,
 	config: ISolanaSdkConfig | undefined
 ): IRaribleInternalSdk {
```

Comment:
Check linter: 91:2 - 'mutableMetadata' is assigned a value but never used (for file packages/solana-sdk/src/sdk/nft/mint/mint.ts)
And also for function getMintNftInstructions amount of arguments so much, you can pass one object with properties, it's more flexible